examples/devopsconsole_v1alpha1_gitsource_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: devopsconsole.openshift.io/v1alpha1
+kind: GitSource
+metadata:
+  name: example-gitsource
+spec:
+  # Add fields here
```

Comment:
What about the rest of the fields?

Reply:
Added other fields to the example - https://github.com/redhat-developer/devopsconsole-operator/pull/7/commits/3c0276e5b85d276681d5829d7fbdee0292856b82

Reply:
Great. Thank you.